### PR TITLE
docs(ru): Add netlify translation and some minor update

### LINKED ISSUE #855: bug: layout shift on clicking and clicking out of the language select dropdown
### Provide environment information

Brave browser on Windows 11

### Describe the bug

On clicking and clicking out of the language select dropdown, there is layout shift.

https://user-images.githubusercontent.com/95541290/203685235-12534d19-2fe6-460d-83a5-0446e134f1ee.mp4

### To reproduce

Replicate what's being done in the video

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Yevhenii-Bakhmat**
Closes 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

* Translated netlify.md
* minor update of typescript.md
* update of T3 collection
---

## Screenshots

💯

### CODE REVIEW

**Comment by Yevhenii-Bakhmat** [comment]
@AmadeusTwi @ronanru

File: www/src/pages/ru/deployment/netlify.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+title: Netlify
+description: Развертывание в Netlify
+layout: ../../../layouts/docs.astro
+---
+
+Netlify это альтернативый провайдер развертывания, похожий на Vercel. Вот [`ajcwebdev/ct3a-netlify`](https://github.com/ajcwebdev/ct3a-netlify) пример репозитория на основе этой документации.
+
+## Зачем развертывать на Netlify
+
+Конвенционная мудрость гласит что Vercel имеет лучшую поддержку Next.js потому что Vercel разрабатывает Next.js. Они заинтересованы в том, чтобы платформа была настроена для оптимальной производительности и DX с Next.js. В большенстве случаев это будет правдой и в отклонении от стандартного пути не будет смысла.
```
Comment (ronanru) [inline www/src/pages/ru/deployment/netlify.md:11]:
```suggestion
Принято считать, что Vercel имеет лучшую поддержку Next.js потому что Vercel разрабатывает Next.js. Они заинтересованы в том, чтобы платформа была настроена для оптимальной производительности и DX с Next.js. В большенстве случаев это будет правдой и в отклонении от стандартного пути не будет смысла.
```

Comment (Yevhenii-Bakhmat) [reply www/src/pages/ru/deployment/netlify.md:11]: fair

File: www/src/pages/ru/deployment/netlify.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+title: Netlify
+description: Развертывание в Netlify
+layout: ../../../layouts/docs.astro
+---
+
+Netlify это альтернативый провайдер развертывания, похожий на Vercel. Вот [`ajcwebdev/ct3a-netlify`](https://github.com/ajcwebdev/ct3a-netlify) пример репозитория на основе этой документации.
+
+## Зачем развертывать на Netlify
+
+Принято считать, что Vercel имеет лучшую поддержку Next.js потому что Vercel разрабатывает Next.js. Они заинтересованы в том, чтобы платформа была настроена для оптимальной производительности и DX с Next.js. В большенстве случаев это будет правдой и в отклонении от стандартного пути не будет смысла.
```
Comment (AmadeusTwi) [inline www/src/pages/ru/deployment/netlify.md:11]:
```suggestion
Принято считать, что Vercel имеет лучшую поддержку Next.js потому что Vercel разрабатывает Next.js. Они заинтересованы в том, чтобы платформа была настроена для оптимальной производительности и DX с Next.js. В большинстве случаев это будет правдой и в отклонении от стандартного пути не будет смысла.
```

File: www/src/pages/ru/deployment/netlify.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+У всех провайдеров развертывания есть преимущества и недостатки, поскольку ни один хост не может иметь лучшую поддержку для всех случаев использования. Например, Netlify разработал свой собственный [пользовательский Next.js runtime](https://github.com/netlify/next-runtime) для Netlify Edge Functions (которые работают на Deno Deploy) и [поддерживают уникальные промежуточные программы для доступа и изменения HTTP-ответов](https://github.com/netlify/next-runtime#nextjs-middleware-on-netlify).
+
+> _Обратите внимание: Для того, чтобы отслеживать статус нестабильных функций Next 13, см. [Использование каталога `app` Next 13 на Netlify](https://github.com/netlify/next-runtime/discussions/1724)._
+
+## Конфигурация проекта
+
+Существует несколько способов настройки инструкций сборки, включая прямое использование Netlify CLI или Netlify dashboard. Хотя это не обязательно, рекомендуется создать и включить файл [`netlify.toml`](https://docs.netlify.com/configure-builds/file-based-configuration/). Это гарантирует, что форкнутые и клонированные версии проекта будут легче повторно развернуть.
```
Comment (AmadeusTwi) [inline www/src/pages/ru/deployment/netlify.md:21]:
```suggestion
Существует несколько способов настройки инструкций сборки, включая прямое использование Netlify CLI или Netlify Dashboard. Хотя это не обязательно, рекомендуется создать и включить файл [`netlify.toml`](https://docs.netlify.com/configure-builds/file-based-configuration/). Это гарантирует, что форкнутые и клонированные версии проекта будут легче повторно развернуть.
```

**Review by AmadeusTwi — COMMENTED** [review]
Left some comments on small typos :).
Other than that — it's good to go.

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**